VerifyEncoding/Test-Encoding.ps1

```diff
@@ -89,14 +100,33 @@ function Test-Encoding
                 continue
             }
 
+            $fileExists = Test-Path -Path $file
```

Comment:
```suggestion
            $fileExists = Test-Path -LiteralPath $file
```

(otherwise, files with weird names containing `[]` will break the script)